performance-test/src/gatling/java/org/opensearch/dataprepper/test/performance/tools/TraceChain.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.dataprepper.test.performance.tools;
+
+import com.google.protobuf.util.JsonFormat;
+import io.gatling.javaapi.core.ChainBuilder;
+import io.gatling.javaapi.core.CoreDsl;
+import io.gatling.javaapi.http.HttpDsl;
+
+import java.util.List;
+
+public class TraceChain {
```

Comment:
Let's call this: TraceChainBuilder.

If we are introducing another `Chain`, let's rename the existing one to `LogChainBuilder`